include/open62541/util.h

```diff
@@ -41,6 +41,8 @@ typedef struct {
     UA_KeyValuePair *map;
 } UA_KeyValueMap;
 
```

Comment:
I see why you want to do this.
But no, please continue to use NULL instead of a sentinel pointer.
This is a public API so we have to protect against NULL anyway.
And it might happen easily that by mistake we load elements into the “const” empty value.

Reply:
What do you think about defining a constant and use that instead (see latest commit). Like this we still have the idiomatic use of that KEYVALUEMAP literal in places where we want to pass NULL. It makes the code easier to read in my opinion. 

User code may or may not use that constant.
```c
#define UA_KEYVALUEMAP_NULL NULL
```